### ISSUE TITLE
Stuck loading air-routes.graphml

### ISSUE THREAD

**Issue by JETopper** [krlawrence/graph#124]
After the command "graph.io(graphml()).readGraph('air-routes.graphml'),"  I get a ParseError at [row,col]:[43,85]
"Message:  Attribute name "data-pjax-transient" associated with an element type "meta" must be followed by the ' = ' character"

The error also occurs if the file is loaded into yEd Graph Editor.

Not sure if I'm doing something wrong.

**Comment by krlawrence** [comment]
Have you tried re-downloading the file just to make sure it is not corrupted for some reason? Are you able to look at the file in a text editor? There should not be any problems with it. I downloaded it and it loaded OK.

**Comment by JETopper** [comment]
Thanks for the fast response.

I did look at the file (using word processor apps) and nothing appears to be out of place (the file wouldn’t load into an XML notepad app I use).  I had the same error message on MAC OS and Windows 7 machines after trying multiple air-route files.  I’m running Java 8 Update 181/Java SDK 8 update 171 (64 bit) on my Windows machine.

I was also able to successfully load other graphml files from the gremlin console.  Not savvy enough to see if I’m doing something wrong downloading or running the file or with my system configuration and will try different approaches.

**Comment by JETopper** [comment]
I figured it out.  I clicked on a hyperlink and saved the graphML file rather than selecting the link and using the download button.  The file as is now works.  Sorry for the trouble and thanks for putting together a great course!



From: Topper, J. Stephen
Sent: Wednesday, September 05, 2018 10:59 AM
To: 'krlawrence/graph' <reply@reply.github.com>
Subject: RE: [krlawrence/graph] Stuck loading air-routes.graphml (#124)

Thanks for the fast response.

I did look at the file (using word processor apps) and nothing appears to be out of place (the file wouldn’t load into an XML notepad app I use).  I had the same error message on MAC OS and Windows 7 machines after trying multiple air-route files.  I’m running Java 8 Update 181/Java SDK 8 update 171 (64 bit) on my Windows machine.

I was also able to successfully load other graphml files from the gremlin console.  Not savvy enough to see if I’m doing something wrong downloading or running the file or with my system configuration and will try different approaches.
